cpp/unreal_plugins/SpCore/Source/SpCore/UserInputComponent.cpp

```diff
@@ -23,7 +23,7 @@ UUserInputComponent::UUserInputComponent()
     SP_LOG_CURRENT_FUNCTION();
 
     PrimaryComponentTick.bCanEverTick = true;
-    PrimaryComponentTick.bTickEvenWhenPaused = false;
+    PrimaryComponentTick.bTickEvenWhenPaused = true;
```

Comment:
I don't think this is an appropriate default. Do you want to be able to apply forces and torques to things even when the game is paused?